docs/services/esi/pagination/cursor-based.md

```diff
@@ -0,0 +1,96 @@
+# Cursor-based Pagination
+
+Cursor-based pagination uses opaque tokens to navigate through datasets, unlike traditional offset-based pagination that uses page numbers.
+Instead of requesting "page 1, page 2, page 3", you use tokens that represent specific positions in the data.
+
+Cursor-based pagination is available on selective few routes, and is slowly being added to others.
+
+## How it works
+
+1. **Initial request**: Make a request without pagination parameters to get the first batch of records.
+2. **Token-based navigation**: Use the returned tokens to move forward (`after`) or backward (`before`) through the dataset.
+3. **Change detection**: The `after` token can be used later to detect new or modified records.
+
+### Request parameters
```

Comment:
Wouldn't hurt to specify these are query parameters, and maybe that they are always also listed on the endpoint documentation itself